tests/testthat/test-print.R

```diff
@@ -180,3 +180,76 @@ testthat::test_that("sas rounding support", {
   expect_identical(export_as_txt(lsting, round_type = "sas"),
                    toString(lsting, round_type = "sas"))
 })
+
+testthat::test_that("listings supports horizontal separators", {
+  result <- as_listing(
+    df = ex_adae,
+    disp_cols = c("ARM"),
+    key_cols = c("USUBJID", "AETOXGR"),
+    add_trailing_sep =  c("ARM", "AETOXGR"), # columns
+    trailing_sep = "k"
+  )
+  result <- head(result, 15)
+
+  expect_equal(
+    sum(
+      sapply(
+        strsplit(toString(result), "\n")[[1]],
+        function(x) {
+          x == paste0(rep(substr(x, 1, 1), nchar(x)), collapse = "")
+        }, USE.NAMES = FALSE
+      )
+    ),
+    9
```

Comment:
This says the number of lines that repeat themselves are 9 ( 8 + 1 that is the standard separator)